src/background.js

```diff
@@ -66,11 +67,22 @@ function init() {
     }).then(function(response) {
       iconData = 'data:image/png;base64,' + btoa(String.fromCharCode(...new Uint8Array(response)));
     }).catch(reason => {
-      log('init getUrl rejected', reason);
+      log('init get iconData rejected', reason);
+    });
+  }
+
+  if (!closeData) {
+    fetch(browser.extension.getURL('icons/stop.svg')).then(response => {
+      return response.arrayBuffer();
+    }).then(function(response) {
+      closeData = 'data:image/svg+xml;base64,' + btoa(String.fromCharCode(...new Uint8Array(response)));
+    }).catch(reason => {
+      log('init get closeData rejected', reason);
     });
   }
 
   browser.storage.local.get().then(items => {
+    delete items.dontShow;
```

Comment:
Hmm, this is the sort of thing I'm getting snagged on for wrapping up the metrics PR. Need to generate & persist a per-client UUID, and didn't see how to do that without reworking the whole storage scheme to move the timers into their own property rather than using the "root" of storage.

Maybe I can just kind of generalize this "delete" thing and have bits that look for alarms filter out known non-alarm keys? (e.g. dontShow, metricsUUID, whatever else might come up)